doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -623,55 +574,176 @@ indexing <https://groups.google.com/group/comp.lang.python/msg/1bf4d925dfbf368?q
 See also `a text by prof.dr. Edsger W.
 Dijkstra <https://www.cs.utexas.edu/users/EWD/transcriptions/EWD08xx/EWD831.html>`__.
```

Comment:
Please delete the entire section beginning "Confusion and flamewars"
```
initial element of a sequence has index 0. Confusion and flamewars arise
```
and ending with 
```
Dijkstra <https://www.cs.utexas.edu/users/EWD/transcriptions/EWD08xx/EWD831.html>`__.
```